lib/hexpm/repository/package_searches/package_searches.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Hexpm.Repository.PackageSearches do
```

Comment:
Let's move this file up one directory to match the rest of the context files in this repo. https://elixiross.slack.com/archives/C02JXUJD4KG/p1717849774114129